Automattic/newspack-listings
# refactor: remove related listings functionality

### All Submissions:






### Changes proposed in this Pull Request:



This PR deprecates the half-implemented Related Listings feature set from https://github.com/Automattic/newspack-listings/pull/58. We may decide to revive this feature in the future, preferably with a closer eye toward reader experience, but for now it's causing more confusion (and [bugs](https://github.com/Automattic/newspack-listings/issues/236)) than doing anything productive.

Closes #236 and #96.
 
### How to test the changes in this Pull Request:

1. Check out this branch.
2. Edit a listing and confirm that the "Related Content" panel and the UI to manage parent/child listings is gone.
3. Make some changes to the listing, publish a new listing, view on the front-end, and do a quick smoke test to make sure that no other listings functionality is missing or broken—this is a pretty self-contained set of features, so it shouldn't affect much of anything.

### Other information:
